apps/storybook/src/ProfilePage.stories.tsx

```diff
@@ -20,7 +20,10 @@ export default {
   title: 'Pages / Profile',
   decorators: [
     LayoutDecorator,
-    makeFlagDecorator('Enable Profile Editing', 'PROFILE_EDITING'),
+    makeFlagDecorator('Enable Works Visibility Editing', 'EDIT_PROFILE_WORKS'),
```

Comment:
Also, editing multiple flags in SB is a bit buggy atm, I've tried fixing before, will look into it again later